_build/data/transport.core.system_settings.php

```diff
@@ -549,7 +549,7 @@
 $settings['feed_modx_news_enabled']= $xpdo->newObject('modSystemSetting');
 $settings['feed_modx_news_enabled']->fromArray(array (
   'key' => 'feed_modx_news_enabled',
-  'value' => '1',
+  'value' => '0',
```

Comment:
I think I rejected this change in previous PR :) Please change it back to `enabled` by default.